python/foglamp/services/south/server.py

```diff
@@ -199,9 +199,10 @@ def run(self):
 
         try:
             await Ingest.stop()
+            _LOGGER.exception('Stopped the Ingest server.')
         except Exception as ex:
             _LOGGER.exception('Unable to stop the Ingest server. %s', str(ex))
-            return
+            # return
```

Comment:
Ingest must stop in its own subprocess, south POLL plugin starts (and does not stop) it even when its get timeout and disconnected